sections/language_model.tex

```diff
@@ -0,0 +1,21 @@
+\section{Language model}\label{sec:lm}
```

Comment:
I think should maybe be moved to after LDA and PR.
However there is one problem with another suggested change that I have, with lda-ir using equation 2 it might make it difficult to move lda-ir to the lda section, if equation 2 has not been used yet.